### PR TITLE
Add documentation for every build server Cake provides a build system integration

### PR DESCRIPTION
**Pull request by pascalberger**
Add documentation for every build server Cake provides a build system integration containing a description mentioning the alias and provider type which is available out of the box, including a simple example. Additionally also lists all extensions which are available for a specific build system.

image

### CODE REVIEW

File: input/docs/integrations/build-systems/github-actions/index.cshtml [hunk 1]
```diff
@@ -0,0 +1,41 @@
+Order: 70
+Title: GitHub Actions
+Description: Integration with GitHub Actions
+---
+
+<h1 id="included-support">Out of the box support</h1>
+
+<p>
+    The <a href="/api/Cake.Common.Build/BuildSystemAliases/6C2FCAF0">AppVeyor alias</a>, which returns an
```
Comment (gep13) [inline input/docs/integrations/build-systems/github-actions/index.cshtml:9]: This should be GitHub Actions, right?

**Review by gep13 — CHANGES_REQUESTED** [review]
Just a minor change...

**Comment by gep13** [comment]
@pascalberger this is a great addition!

File: input/docs/integrations/build-systems/github-actions/index.cshtml (hunk 1 above)
Comment (pascalberger) [reply input/docs/integrations/build-systems/github-actions/index.cshtml]: Fixed

**Review by gep13 — APPROVED** [review]
LGTM!

**Comment by gep13** [comment]
@pascalberger your changes have been merged, thanks for your contribution 👍

**Pull request merged**